blueprints/tools/blueprint_crs_converter.py

```diff
@@ -9,30 +10,18 @@
 import flask_cors
 import werkzeug
 from opengeodeweb_back import geode_functions
-import json
-
-with open("blueprints/tools/crs_converter_allowed_files.json", "r") as file:
-    crs_converter_allowed_files_json = json.load(file)
 
-with open("blueprints/tools/crs_converter_allowed_objects.json", "r") as file:
-    crs_converter_allowed_objects_json = json.load(file)
-
-with open("blueprints/tools/crs_converter_convert_file.json", "r") as file:
-    crs_converter_convert_file_json = json.load(file)
+crs_converter_routes = flask.Blueprint("crs_converter_routes", __name__)
+flask_cors.CORS(crs_converter_routes)
 
+with open("blueprints/tools/crs_converter_versions.json", "r") as file:
```

Comment:
Maybe a global question to answer all together : where should we import the json schema? At the beginning of each file or juste before its route definition?

Reply:
I agree on this one